### PR TITLE
Set up application structure for API

### PR DESCRIPTION
**Pull request by Alex-Bellamy**
### User Story
As a development team
In order to be able to start adding features
We would like to have an basic application structure
## Changes proposed in this pull request:
* Rails scaffold
* Adding rack-cors gem and configuration
## Packages/Gems added
- Initial rails default gems

### CODE REVIEW

**Review by Odeane — COMMENTED** [review]
looks great...but it still says 'WIP'....so unfortunately it cannot be merged 😏

File: config/application.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  class Application < Rails::Application
+    config.generators do |generate|
+      generate.helper false
+      generate.assets false
+      generate.view_specs false
+      generate.helper_specs false
+      generate.routing_specs false
+      generate.controller_specs false
+  end
+
```
Comment (emtalen) [inline config/application.rb]: These generators has to do with Rspec and should be added when you configure that, not in the scaffold.

**Review by Cma-B — APPROVED** [review]
good job , just I guess ruby version should be changed to 2.5.1!?

**Review by emtalen — COMMENTED** [review]
You need to add and configure rack-cors.

File: config/application.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      allow do
+        origins '*'
+        resource '*', 
+          headers: :any, 
+          methods: %i[get post put delete],
+          expose: %w(access-token expiry token-type uid client),
+          max_age: 0
+      end
+  end
+end
```
Comment (emtalen) [inline config/application.rb:32]: It's an end missing, make sure to format correctly and close every do/class/module (don't copy paste)

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+ruby-2.6.3
```
Comment (emtalen) [inline .ruby-version]: you need to change the ruby version here as well

**Pull request merged**